packages/grid/src/RowSelectionCheckboxColumn.tsx

```diff
@@ -10,13 +12,27 @@ export type RowSelectionCheckboxColumnProps<T> = Omit<
 export function RowSelectionCheckboxColumn<T>(
   props: RowSelectionCheckboxColumnProps<T>
 ) {
+  const { selectRows } = useSelectionContext();
+
+  const onKeyDown = (
+    event: KeyboardEvent<HTMLDivElement>,
+    rowIndex: number
+  ) => {
+    if (event.key === " ") {
+      selectRows(rowIndex, false, true);
```

Comment:
Can you refactor this to a `selectRows({rowIndex, whateverThisIs: false, andThat: true});`? As soon as I have more than one param, I usually refactor to object param, especially when I have boolean flags.

Reply:
Done